doc/src/partials/buttons.vue

```diff
@@ -5,19 +5,19 @@
 .section
   title-link Buttons
   .section__content
-    p The button component allows to easily toggle classes. It supports the 
-      a(v-link.literal='/ripple-effect')
+    p The button component allows to easily toggle classes. It supports the
```

Comment:
Careful about the extra whitespaces a the end of lines on the doc. They're actually needed. They should be replaced by `&nbsp;`